testnet/stacks-node/src/nakamoto_node/sign_coordinator.rs

```diff
@@ -196,9 +200,10 @@ impl SignCoordinator {
         reward_set: &RewardSet,
         reward_cycle: u64,
         message_key: Scalar,
-        aggregate_public_key: Point,
+        aggregate_public_key: Option<Point>,
```

Comment:
Let's just remove the aggregate public key.  It's not going to be used for some time.

Reply:
Ok, can do. For things like `begin_sign_v1`, should we just use `Point::new()` with a TODO?